data/json/mapgen/hunting_lodge.json

```diff
@@ -0,0 +1,180 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "lodge_ground1", "lodge_ground2" ],
```

Comment:
"om_terrain": [ [ "lodge_ground1", "lodge_ground2" ] ],

double brackets should fix your problem (for all the merged maps).